examples/project-commands/README.md

```diff
@@ -0,0 +1,42 @@
+# project-commands
+
+> **Warning**
+> If you copy the flake.nix remember to `git add [-N|--intent-to-add] flake.nix`, otherwise it won't work
+
+This example shows how to create scripts for your project, by leveraging [mission-control](https://github.com/Platonic-Systems/mission-control)
+
+This is a **potential** alternative to:
+
+- Using a `Makefile` to manage your project's scripts
+- Using the popular [Scripts To Rule Them All](https://github.com/github/scripts-to-rule-them-all) approach (having a `scripts/` folder)
```

Comment:
```suggestion
- Using the popular [Scripts To Rule Them All](https://github.com/github/scripts-to-rule-them-all); a naming convention for a `scripts/` directory
```

A bit more info, and folder -> directory, which is the term we generally use in and around Nix.